### PR TITLE
Test push notifications

### PR DESCRIPTION
**Pull request by timarney**
Updating unit tests for sending reminders + exposed

### CODE REVIEW

File: src/services/ExposureNotificationService/ExposureNotificationService.ts [hunk 1]
```diff
@@ -180,6 +180,23 @@ export class ExposureNotificationService {
...
   }
 
+  public isReminderNeeded(exposureStatus: ExposureStatus) {
+    if (exposureStatus.type !== ExposureStatusType.Diagnosed) {
+      return false;
+    }
+
+    if (exposureStatus.needsSubmission) {
+      return true;
+    }
```
Comment (smcmurtry) [inline src/services/ExposureNotificationService/ExposureNotificationService.ts:190]: Comparing this to the old logic, I think it should be:
```
if (!exposureStatus.needsSubmission) {
  return false;
}
```

**Review by smcmurtry — CHANGES_REQUESTED** [review]
Tests look good! Just one bit of logic I noticed needs fixing.

File: src/services/ExposureNotificationService/ExposureNotificationService.ts (hunk 1 above)
Comment (timarney) [reply src/services/ExposureNotificationService/ExposureNotificationService.ts:190]:
```
if (
      exposureStatus.type === ExposureStatusType.Diagnosed &&
      exposureStatus.needsSubmission &&
      (!exposureStatus.uploadReminderLastSentAt ||
        minutesBetween(new Date(exposureStatus.uploadReminderLastSentAt), new Date()) >
          MINIMUM_REMINDER_INTERVAL_MINUTES)
    ) {
```

**Review by smcmurtry — APPROVED** [review]
Looks good!

**Pull request merged**